src/generic/spinctlg.cpp

```diff
@@ -476,6 +477,28 @@ void wxSpinCtrlGenericBase::OnTextChar(wxKeyEvent& event)
         DoSendEvent();
 }
 
+void wxSpinCtrlGenericBase::OnMouseWheel(wxMouseEvent& event)
+{
+    double value = m_value;
+
+    if( event.GetWheelRotation() > 0 )
+    {
+        value += m_increment;
+    }
+    else
```

Comment:
I'm not sure if it can cause actual problems here, but the case of `.GetWheelRotation()` returning zero is not handled. At many other places within wx's source the case of zero is checked.

Also for the whole function, it should be checked that `.GetWheelAxis() == wxMOUSE_WHEEL_VERTICAL`, otherwise also horizontal scrolling modifies the value. Not dangerous, but a little unexpected maybe.

Reply:
Hi lanurmi,

thanks for your review.

I see your point regarding wxMOUSE_WHEEL_VERTICAL. As for GetWheelRotation() returning zero, I don't see a concrete issue there, but for the sake of completeness, I think the case should still be explicitly ignored.